build.gradle

```diff
@@ -41,6 +48,9 @@ if (!project.hasProperty('version') || project.version == 'unspecified') {
     else {
         project.version = tagStr
     }
+    if (!project.hasProperty('useHadoop2')) {
+      project.version = project.version + "-hadoop1"
```

Comment:
Is that necessary? We need "-hadoop1" to differentiate between Hadoop-1 and Hadoop-2 artifacts. Beyond that the POM dependencies should take care of specifying the version of Hadoop1 used.